trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts

```diff
@@ -236,7 +237,7 @@ export const mapFeeRecordPaymentGroupsToPaymentDetailsViewModel = (feeRecordPaym
     {},
   );
 
-  return feeRecordPaymentGroups.reduce((paymentDetails, { feeRecords, paymentsReceived }) => {
+  return feeRecordPaymentGroups.reduce((paymentDetails, { feeRecords, paymentsReceived, status }) => {
```

Comment:
Can we rename `status` to `feeRecordPaymentGroupStatus`, so the return object is clean.

Reply:
I have performed the rename in the destructuring of the type, but I would prefer not to rename the `FeeRecordPaymentGroup` `status` property itself. The reason to call it `feeRecordPaymentGroupStatus` in this context is that the individual payment does not have a "status", but the group it is in does